### PR TITLE
Adelle/commenting

### PR DESCRIPTION
**Pull request by Nagel29**
# Description
This branch adds the ability for users to add and delete comments for individual games on the game's details page. It also adds the styling for the form and comments sections.

# Fixes # (issue)
#38 

# Type of change

- [ ]  Bug fix (non-breaking change which fixes an issue)
 
- [x]  New feature (non-breaking change which adds functionality)

- [ ]  Breaking change (fix or feature that would cause existing functionality not to work as expected)

- [ ] This change requires a documentation update

-  [x] Styling

- [ ] Refactor

- [ ] Testing

# Where were the changes?
App.tsx, CommentCard.tsx, CommentCard.css, Comments.tsx, Comments.css, Details.tsx, Details.css, Form.tsx, Form.css

# Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [ ]  I have commented my code, particularly in hard-to-understand areas (if applicable)
- [ ]  I have made corresponding changes to the documentation (if applicable)
- [x]  My changes generate no new warnings

# Contributors (paired programming):
@Courtney-Lippman @Adelle-Pitsas @knpausch @dgubko

### CODE REVIEW

File: src/components/CommentCard/CommentCard.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import './CommentCard.css';
+import { CgTrash } from "react-icons/cg"
+
+const CommentCard = ({commentId, comment, name, gameId, deleteComment }: {commentId: number, comment: string, name: string, gameId: string, deleteComment: (commentId: number, id: string) => void}) => {
+  return (
+    <div className="comment-container">
+      <p>{name} said: "{comment}"</p>
+      <p onClick={() => deleteComment(commentId, gameId)}>{<CgTrash className="delete"/>}</p>
```
Comment (Courtney-Lippman) [inline src/components/CommentCard/CommentCard.tsx:8]: Great use of a react icon!

File: src/components/Comments/Comments.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Review } from "../../interfaces"
+import CommentCard from '../CommentCard/CommentCard'
+import './Comments.css';
+
+const Comments = ({ comments, id, deleteComment }: {comments: Review[], id: string, deleteComment: (commentId: number, id: string) => void}) => {
```
Comment (Courtney-Lippman) [inline src/components/Comments/Comments.tsx:5]: The types look good here!

**Review by Courtney-Lippman — APPROVED** [review]
Everything looks good! The styling looks great!

**Review by dgubko — APPROVED** [review]
Looks great, I like this new commenting feature!

**Pull request merged**